frontend/src/components/Profile/MyAchievements.vue

```diff
@@ -3,41 +3,111 @@
     <v-card-title class="white--text orange darken-4" align="center">
       Achievements
     </v-card-title>
-    <v-spacer></v-spacer>
-    <v-card>
-      achievements list
-    </v-card>
+    <v-divider></v-divider>
+    <v-row>
+      <div v-for="tabAch in tabAchs" :key="tabAch.id">
+        <v-col align="center">
+        <img v-if="tabAch.img" :src="tabAch.img" :title="tabAch.description" /></v-col>
+        <v-col align="center">
+        <p class="achievementsName">{{ tabAch.name }}</p>
+      </v-col>
+      </div>
+      <p class="pt-10 achievementsName" v-if="!tabAchs.length" >
+        No achievement obtained yet.
+      </p>
+    </v-row>
   </v-card>
 </template>
 
 <script lang="ts">
 import axios from 'axios';
 import { defineComponent } from 'vue';
 import { mapGetters } from 'vuex';
+import imageAch1 from '@/assets/achievements/ping.png';
+import imageAch2 from '@/assets/achievements/pong.png';
+import imageAch3 from '@/assets/achievements/addict.png';
+import imageAch4 from '@/assets/achievements/beginnersLuck.png';
+import imageAch5 from '@/assets/achievements/playerDiff.png';
+import imageAch6 from '@/assets/achievements/mastery.png';
+import imageAch7 from '@/assets/achievements/king-pong.png';
+
+interface Achievements {
+  idOther: number,
+  tabAchs: { id: number, img: string, name: string, description: string } [],
+}
 
 export default defineComponent({
-  data: () => ({
-    idOther: 0,
-  }),
+  data(): Achievements {
+    return {
+      idOther: 0,
+      tabAchs: [],
+    };
+  },
   methods: {
     ...mapGetters(['id']),
+    setAchievementImg() {
```

Comment:
There is probably a more elegant way of doing this but I'm still going to approve the pr because from what i saw everything works as intended